docs/development.md

```diff
@@ -7,27 +7,30 @@ See [Getting Started with the Raspberry Pi Pico](https://rptl.io/pico-get-starte
 
 Most of this will be parroting the above linked PDF from the Raspberry Pi Foundation.
 
-### Windows Setup
+If you are very new to GP2040-CE development and working on a Windows machine, You can refer to the 'Windows Setup (Pi Pico SDK Installer)' section in this page.
```

Comment:
Change the wording to this:

For GP2040-CE development on the Windows platform using the Pico SDK Installer, please skip ahead to the 'Windows Setup (Pico SDK Installer)' section on this page.

### Windows Setup (Manual)

Reply:
Changed